.github/workflows/compat_minimal.yml

```diff
@@ -64,7 +64,14 @@ jobs:
       - shell: bash -el {0}
         run: ./tools/github_actions_infos.sh
         name: 'Show infos'
+      - uses: actions/cache@v2
+        id: cache-dataset
+        with:
+          path: /home/runner/mne_data
+          key: ${{ runner.os }}-dataset
+        name: 'Cache dataset'
       - shell: bash -el {0}
+        if: steps.cache-dataset.outputs.cache-hit != 'true'
```

Comment:
We should probably always try to download it, it should be a no-op if the files already exist. We just need to make sure that the version-number checking actually works (e.g., if we specify testing dataset version 0.110 but 0.109 is found in version.txt, that the new one gets downloaded)